_posts/en/newsletters/2019-12-18-newsletter.md

```diff
@@ -0,0 +1,238 @@
+---
+title: 'Bitcoin Optech Newsletter #77'
+permalink: /en/newsletters/2019/12/18/
+name: 2019-12-18-newsletter
+slug: 2019-12-18-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter announces the release of LND 0.8.2-beta, requests
+help testing the latest C-Lightning release candidate, discusses
+widespread support for basic multipath payments in LN, provides an update on
+bech32 error detection reliability, summarizes updates to the proposed
+`OP_CHECKTEMPLATEVERIFY` opcode, and links to a discussion about the impact
+of eclipse attacks on LN channels.  Also included is our regular section
+about notable changes to popular Bitcoin infrastructure projects. <!--
+FIXME: also mention other sections if schmidty decides to write them -->
+
+{% comment %}<!-- include references.md below the fold but above any Jekyll/Liquid variables-->{% endcomment %}
+{% include references.md %}
+
+## Action items
+
+- **Upgrade to LND 0.8.2-beta:** this [release][lnd 0.8.2-beta] contains
+  several bug fixes and minor UX improvements, most notably for the
+  recovery of static channel backups.
+
+- **Help test C-Lightning 0.8.0 RCs:** the [release candidate][cl 0.8.0]
+  for the next version of C-Lightning switches the default network to
+  mainnet instead of testnet (see [Newsletter #75][news75 cl mainnet])
+  and adds support for basic multipath payments (described below) among
+  many other features and bug fixes.
+
+## News
+
+- **LN implementations add multipath payment support:** after a
+  significant amount of discussion and development, all three LN
+  implementations tracked by Optech have now added basic support for
+  [multipath payments][topic multipath payments].  A multipath payment
+  consists of multiple LN payments routed over different paths which may
+  all be claimed at the same time by the receiver.  This provides a
+  major upgrade to the usability of LN by allowing users with their
```

Comment:
s/allowing users with their balance split across multiple channels to spend/allowing users, with their
  balance split across multiple channels, to spend